queue-staging.tf

```diff
@@ -0,0 +1,104 @@
+module "queue-staging-namespace" {
```

Comment:
you can use the same namespace but create a different queue

Reply:
should not we clearly split staging and non staging?

Reply:
If we have a good reason then yes otherwise it is not needed